servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1957,22 +1957,7 @@ void RenderForwardClustered::_render_scene(RenderDataRD *p_render_data, const Co
 			RD::get_singleton()->draw_command_begin_label("Setup Sky");
 
 			// Setup our sky render information for this frame/viewport
-			if (is_reflection_probe) {
-				Vector3 eye_offset;
-				Projection correction;
-				correction.set_depth_correction(true);
-				Projection projection = correction * p_render_data->scene_data->cam_projection;
-
-				sky.setup_sky(p_render_data->environment, rb, *p_render_data->lights, p_render_data->camera_attributes, 1, &projection, &eye_offset, p_render_data->scene_data->cam_transform, projection, screen_size, Vector2(0.0f, 0.0f), this);
-			} else {
-				Projection projection = p_render_data->scene_data->cam_projection;
-				if (p_render_data->scene_data->cam_frustum) {
-					// Sky is drawn upside down, the frustum offset doesn't know the image is upside down so needs a flip.
-					projection[2].y = -projection[2].y;
-				}
-
-				sky.setup_sky(p_render_data->environment, rb, *p_render_data->lights, p_render_data->camera_attributes, p_render_data->scene_data->view_count, &projection, p_render_data->scene_data->view_eye_offset, p_render_data->scene_data->cam_transform, projection, screen_size, p_render_data->scene_data->taa_jitter, this);
-			}
+			sky.setup_sky(p_render_data, screen_size, p_render_data->scene_data->taa_jitter);
```

Comment:
Seeing you're passing `p_render_data` here, does it make sense to pass `p_render_data->scene_data->taa_jitter` here as it will be accessible from its source?

Reply:
Yep!